plugins/websocket/src/Client.cpp

```diff
@@ -76,8 +76,10 @@ Client::metadata() noexcept
 	try
 	{
 		using namespace boost::json;
-		const value jsonValue{
-		 parse(string_view{reinterpret_cast<const char*>(buffer.data()), buffer.size()})};
+		parser jsonParser;
+		jsonParser.write(string_view{reinterpret_cast<const char*>(buffer.data()), buffer.size()});
+		value jsonValue = jsonParser.release();
+
 		const object& jsonObject{jsonValue.as_object()};
```

Comment:
Here I wouldn't use `string_view` if there is an overload that accepts pointer + size as direct parameters. It seems possible to contract the 2nd and 3rd instructions somehow but I don't understand the reference/value semantics these functions have so the final call is yours.